SRTNet.cpp

```diff
@@ -220,6 +220,7 @@ void SRTNet::serverEventHandler(bool singleClient) {
             }
         }
 
+        std::lock_guard<std::mutex> lock(mClientListMtx);
```

Comment:
Put the whole thing inside brackets to make sure next edit here will not keep the look longer than necessary.

Reply:
Good suggestion! Fixed.